wiki/Music_theory/Syncopation/en.md

```diff
@@ -0,0 +1,7 @@
+---
+stub: true
+---
+
+# Syncopation
+
+**Syncopation** is the accenting of a note which would usually not be accented. The [time signature](/wiki/Music_theory/Time_signature) of a piece of music gives an indication of a regular [rhythm](/wiki/Music_theory/Rhythm) pattern of strong and weak beats. A syncopated rhythm goes against this expectation by putting the accent on weak beats, thereby creating counterflow.
```

Comment:
You could additionally explain it more simply with something along the lines of "placing notes off-beat, i.e. on 'red timeline ticks'"

Reply:
added